lib/vdsm/virt/vmdevices/storage.py

```diff
@@ -233,7 +234,7 @@ def __init__(self, log, **kwargs):
         if not hasattr(self, 'vm_custom'):
             self.vm_custom = {}
         self._monitorable = True
-        self._threshold_state = BLOCK_THRESHOLD.UNSET
+        self.threshold_state = BLOCK_THRESHOLD.UNSET
```

Comment:
Unrelated change. Any reason to make it public?

Reply:
and rest of the code still uses `self._threshold_state`

Reply:
Change is related - threshold_state is already public. When use use the public accessor
we change it via @threshold_state.setter which set exceeded_time to None.

When we set drive threshold we use:

   drive.threshold_state = value

So each time we set it we clear the exceeded time. The only place when we set it using
the private _threshold_state is in on_block_threshold(), and there we set exceeded_time.

This is little bit delicate, but matches the current usage.